app/src/main/AndroidManifest.xml

```diff
@@ -194,18 +200,36 @@
             android:theme="@style/GreyBackgroundTheme"
             android:windowSoftInputMode="adjustResize" />
 
+
+        <activity
+            android:name=".ui.safe.connect.Connect2FaStartActivity"
+            android:screenOrientation="portrait"
+            android:theme="@style/WhiteTheme" />
+
+        <activity
+            android:name=".ui.safe.recover.extension.Replace2FaStartActivity"
+            android:screenOrientation="portrait"
+            android:theme="@style/WhiteTheme" />
+
         <activity
-            android:name=".ui.safe.recover.extension.ReplaceAuthenticatorSubmitActivity"
+            android:name=".ui.safe.connect.Remove2FaStartActivity"
+            android:screenOrientation="portrait"
+            android:theme="@style/WhiteTheme" />
+
+
+        <activity
+            android:name=".ui.safe.recover.extension.ReplaceAuthenticatorRecoveryPhraseActivity"
```

Comment:
Should we rename the `extension` to `authenticator`?

Reply:
yes, makes sence